jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/exceptions/ApiResponseExceptionHandlerTest.java

```diff
@@ -46,8 +46,8 @@ void handlePersistenceException_badRequestWithErrorDataIsReturned() {
         var apiResponseExceptionHandler = new ApiResponseExceptionHandler();
         var exception = new PersistenceException(ERROR_MESSAGE);
         var response = apiResponseExceptionHandler.handlePersistenceException(exception);
-        var msg = "There was an error persisting data.";
-        assertResponseData(response, msg, HttpStatus.BAD_REQUEST);
+        var msg = "Server Error: Unable to process the request.";
```

Comment:
the test name here needs to be updated to reflect a server error instead of bad request.